static/app/views/replays/details.tsx

```diff
@@ -122,6 +125,10 @@ const ReplayLayout = styled(Layout.Main)`
   }
 `;
 
+const Side = styled(Layout.Side)`
+  padding-bottom: ${space(2)};
```

Comment:
Actually, is this compensation because the `const Panel ...` has `margin-bottom: 0;` applied?

Maybe we don't need either of those css properties, and things will line up be default. Fewer custom stuff is better.